custom/conf/app.example.ini

```diff
@@ -2583,6 +2583,8 @@ LEVEL = Info
 ;ENDLESS_TASK_TIMEOUT = 3h
 ;; Timeout to cancel the jobs which have waiting status, but haven't been picked by a runner for a long time
 ;ABANDONED_JOB_TIMEOUT = 24h
+;; Strings committers can place inside a commit message to skip executing the corresponding actions workflow
+;SKIP_WORKFLOW_STRINGS = "[skip ci]", "[ci skip]", "[no ci]", "[skip actions]", "[actions skip]"
```

Comment:
That's not INI syntax .... IMO it should be:


```
;SKIP_WORKFLOW_STRINGS = "[skip ci],[ci skip],[no ci],[skip actions],[actions skip]"

or simply:

;SKIP_WORKFLOW_STRINGS = [skip ci],[ci skip],[no ci],[skip actions],[actions skip]
```

Reply:
I guess I'll never be a friend of ini 😆 Corrected

Reply:
It's still not fixed.

Reply:
Oops, forgot to push....